niflow_manager/data/templates/base/spec.yml

```diff
@@ -0,0 +1,49 @@
+# the build part will be used to create an image with `nfm build`
+# required_env is the required entry and it specifies the environment needed to run the analyses,
+# required_env will be also used as an additional environment during testing with `nfm test`
+# entrypoint - an optional entrypoint for the container (niflow-{ORGANIZATION}-{WORKFLOW} is the default value)
+build:
+  # example below, see full Neurodocker spcification: https://github.com/ReproNim/neurodocker#supported-software
+  required_env:
+    base:
+      image: debian:stretch
+      pkg-manager: apt
+    miniconda:
+      conda_install: [python=3.7, nipype]
+#    fsl:
+#      version: 5.0.10
+#    afni:
+#      version: latest
+#
+# optional entrypoint, for python, the default value is
+#  entrypoint: /opt/miniconda-latest/envs/testkraken/bin/niflow-{ORGANIZATION}-{WORKFLOW}
```

Comment:
i would remove the `testkraken` reference. also the `entrypoint` should be in `PATH`, so an absolute path should not be necessary.

Reply:
by "should" you mean that neurodocker should add it or I should add this to the `PATH`? I was surprised that it doesn't work without the absolute path, but it didn't

Reply:
if it doesn't work without absolute path, i guess you can leave it with the full path (with the updated env name).